.github/workflows/CI.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.10', '3.11', '3.12']
+        python-version: ['3.10', '3.12', '3.13']
```

Comment:
Did we drop Python 3.11 support?

Reply:
It was discussed in yesterday's aerotools meeting. We are removing it for efficiency reasons as 3.11 is used by most people during work locally: https://github.com/metno/pyaerocom/issues/1436#issuecomment-2531954690

Reply:
PS: By habit I rebased and force pushed the branch so you will have to delete and pull the branch again. I'll try to avoid this for code reviews in the future.